plugins/linux-pipewire/pipewire.h

```diff
@@ -22,16 +22,31 @@
 
 #include <obs-module.h>
 
-typedef struct _obs_pipewire_data obs_pipewire_data;
+#include <pipewire/pipewire.h>
 
-void *obs_pipewire_create(int pipewire_fd, int pipewire_node);
-void obs_pipewire_destroy(obs_pipewire_data *obs_pw);
+typedef struct _obs_pipewire obs_pipewire;
+typedef struct _obs_pipewire_stream obs_pipewire_stream;
```

Comment:
I think that names that begin with underscore are reserved, though I might be misunderstanding the "reserved names" section of the C standard. @kkartaltepe may understand it more than I.

Reply:
I'm not aware of any kind of name reservation mechanism in C. The C language has reserved keyword (if, else, while, ...), and the standard types and functions of `libc` must not be overriden by application code, but I don't think there's anything preventing applications from defining structs like here.

Reply:
See the following regarding identifiers that begin with an underscore:
* https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html
* https://en.cppreference.com/w/c/language/identifier

I generally would opt to just avoid starting any identifier with an underscore just to avoid having to wonder if it's acceptable according to the standard.